package.json

```diff
@@ -36,7 +36,7 @@
     "cookie-parser": "^1.4.6",
     "express": "^4.17.3",
     "express-validator": "^6.14.0",
-    "govuk-frontend": "^4.0.1",
+    "govuk-frontend": "^4.6.0",
```

Comment:
This will need careful testing for both registration and update.

Reply:
Heads up raised on #roe-scrum-teams.

Reply:
Do we need to update package-lock.json?

Reply:
Yeah, will need the package-lock changes for this upgrade. @cdrainCH this is the one time you should push that :)

Reply:
Need to be on node 16 and make sure that only the govuk-frontend related info/sections are changed. If it looks like a huge diff then it's bad. Generate with "npm install".